### ISSUE TITLE
primitive array lists

### ISSUE THREAD

**Issue by dmk23** [OpenHFT/Koloboke#41]
I've been benchmarking Koloboke against several alternatives and have been very pleased with performance. Would like to standardize all our code on it but see that right now there are no primitive array lists (e.g like IntArrayList from Mahout/Trove).

Are there any plans to include such functionality?

**Comment by leventov** [comment]
Certainly, we are going to add lists, and many more containers, as a part of this project: https://github.com/OpenHFT/Koloboke/wiki/Koloboke:-roll-the-collection-implementation-with-features-you-need

Good news: this project is already in it's progress. Bad news: it is in a very slow progress. If you see a business value in using the coolest collection library, consider your company for sponsoring the development.

**Comment by dmk23** [comment]
Thanks for the link - this roadmap is very helpful.

At the moment we are not ready to sponsor any specific projects but will keep such opportunities in mind
